### PR TITLE
add cloudant case in integration test

### LINKED ISSUE #473: Use Case - cloudant feed
Please add a realistic cloudant use case which can than be used as part of integration test. We don't have any manifest/deployment files for cloudant (showing how to configure credentials: username, password, url, etc) in our repo.

Priority: High
Category: use case, enhancement

### PR DESCRIPTION
**Pull request by daisy-ycguo**
Close #473 

This integration test case will deploy follow items to OpenWhisk:
```
Entities in namespace: <namespace>
packages
/<namespace>/openwhisk-cloudant                            private
/<namespace>/cloudant-sample                               private
actions
/<namespace>/cloudant-sample/process-change-cloudant-sequence private sequence
/<namespace>/cloudant-sample/process-change                private nodejs:6
triggers
/<namespace>/data-inserted-trigger                         private
rules
/<namespace>/log-change-rule                               private
```

### CODE REVIEW

File: tests/src/integration/cloudant/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+package:
+    name: cloudant-sample
+    dependencies:
+        # binding cloudant package named openwhisk-cloudant
+        openwhisk-cloudant:
+            location: /whisk.system/cloudant
+            inputs:
+                username: $CLOUDANT_USERNAME
+                password: $CLOUDANT_PASSWORD
+                host: ${CLOUDANT_USERNAME}.cloudant.com
```
Comment (pritidesai) [inline tests/src/integration/cloudant/manifest.yaml]: @daisy-ycguo do we support this structure, i have tried many times and it does not replace env. variable $CLOUDANT_USERNAME in ${CLOUDANT_USERNAME}.cloudant.com.

Comment (daisy-ycguo) [reply tests/src/integration/cloudant/manifest.yaml]: We don't support this. This manifest file doesn't work.

Comment (mrutkows) [reply tests/src/integration/cloudant/manifest.yaml]: @pritidesai daisy opened another issue here to address this as a new feature... issue https://github.com/apache/incubator-openwhisk-wskdeploy/issues/501

**Comment by daisy-ycguo** [comment]
This PR depends on PR #513.
It must be merged after PR #513.

**Comment by daisy-ycguo** [comment]
Please review and merge it.

**Pull request merged**